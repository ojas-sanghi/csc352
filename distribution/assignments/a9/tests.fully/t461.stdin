94
74
78
-
*
23
57
*
44
53
+
*
+
