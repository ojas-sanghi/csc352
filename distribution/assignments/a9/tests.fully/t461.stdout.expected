94 74 78 - * 23 57 * 44 53 + * +
((94*(74-78))+((23*57)*(44+53)))
